omod/src/main/java/org/openmrs/module/dhisreport/web/controller/Dhis2ServerController.java

```diff
@@ -6,7 +6,7 @@
  *  DHIS2 Reporting module is free software: you can redistribute it and/or modify
  *  it under the terms of the GNU General Public License as published by
  *  the Free Software Foundation, either version 3 of the License, or
- *  (at your option) any later version.
+ *  (at your option) any later version.
 
```

Comment:
these two lines are exactly same but still Github is showing that the change. I tried to 4-5 times to make it right but didn't quite make it. Please let me know how i can fix this.